packages/web-client/app/components/card-pay/deposit-workflow/transaction-setup/token-option/index.hbs

```diff
@@ -1,33 +1,26 @@
-<label class={{cn "token-option" token-option--checked=@checked}} ...attributes>
+<label
+  class={{cn "token-option" token-option--checked=@checked}}
+  data-test-option={{@token.symbol}}
+  ...attributes
+>
   <Input
     name="deposit-token"
     class={{cn "token-option__input" token-option__input--checked=@checked}}
     @type="radio"
     @checked={{@checked}}
     {{on "input" @onInput}}
   />
-  <div class="token-option__currency">
-    {{svg-jar @tokenIcon class="token-option__icon" role="presentation" width="40" height="40"}}
-    <div>
-      <div class="token-option__title" data-test-option={{@tokenSymbol}}>
-        {{@tokenSymbol}}
-      </div>
-      <div class="token-option__desc">
-        {{@tokenDescription}}
-      </div>
-    </div>
-  </div>
-  <div>
-    <div class="token-option__balance-header">Balance</div>
-    <div class="token-option__balance" data-test-balance={{@tokenSymbol}}>
-      {{@balance}} {{@tokenSymbol}}
-    </div>
-    <div class="token-option__usd-balance" data-test-usd-balance={{@tokenSymbol}}>
-      {{#if @balanceInUsd}}
-        ≈ $ {{@balanceInUsd}} USD
-      {{else}}
-        <Boxel::LoadingIndicator class="token-option__loading-indicator"/>
-      {{/if}}
-    </div>
-  </div>
+  <CardPay::BalanceDisplay
+    @size="large"
+    @icon={{@token.icon}}
+    @name={{@token.symbol}}
+    @text={{@token.description}}
+  />
+  <CardPay::BalanceDisplay
+    @label="Balance"
+    @balance={{@balance}}
+    @symbol={{@token.symbol}}
+    @displayUsdBalance={{true}}
+    data-test-balance={{@token.symbol}}
+  />
```

Comment:
Using the component this way seems to tie us to a certain implementation of a balance display. I think keeping the original implementation is better here because it works, and we don't need to think about it, and come back to change this component if/when we decide we want a new implementation of balance display.

Reply:
Yeah I was undecided about this one, so your opinion helps.